ols/src/llms/providers/provider.py

```diff
@@ -353,8 +369,8 @@ def _construct_httpx_client(
                 )
                 verify = custom_context
             if use_async:
-                return httpx.AsyncClient(verify=verify)
-            return httpx.Client(verify=verify)
+                return httpx.AsyncClient(verify=verify, proxies=proxy)
+            return httpx.Client(verify=verify, proxies=proxy)
```

Comment:
I assume it configures proxy for https, not http, right?

Reply:
yes, it is for HTTPS proxy. 
any modification required on these 2 lines?